src/DssSpell.sol

```diff
@@ -43,8 +49,34 @@ contract DssSpellAction is DssAction {
     uint256 internal constant WAD     = 10 ** 18;
     uint256 internal constant RAY     = 10 ** 27;
```

Comment:
We can comment out the `MILLION`, `WAD`, and `RAY` lines to reduce gas costs on deploy as they won't be needed.

Reply:
I thought constants were subsituted during compilation and didn't take up any state? Either way I will remove since unused